refarch-gateway/README.md

```diff
@@ -41,16 +41,18 @@ Beside the default behaviour there are some special route prefixes which are han
 
 ## Configuration
 
-| Var                                                      | Description                                        | Example                                                                 |
-|----------------------------------------------------------|----------------------------------------------------|-------------------------------------------------------------------------|
-| `SPRING_PROFILES_ACTIVE`                                 | See profiles                                       | `local,hazelcast-local`                                                 |
-| `SPRING_CLOUD_GATEWAY_ROUTES_<index>_ID`                 | Id of a route definition.                          | `backend`                                                               |
-| `SPRING_CLOUD_GATEWAY_ROUTES_<index>_URI`                | The uri to route to if this route matches.         | `http://backend-service:8080/`                                          |
-| `SPRING_CLOUD_GATEWAY_ROUTES_<index>_PREDICATES_<index>` | Route predicates i.e. matcher.                     | `Path=/api/backend-service/**`                                          |
-| `SPRING_CLOUD_GATEWAY_ROUTES_<index>_FILTERS_<index>`    | List of filters applied to the route.              | `RewritePath=/api/backend-service/(?<urlsegments>.*), /$\{urlsegments}` |
-| `ALLOWED_ORIGINS_PUBLIC` (optional)                      | List of urls allowed as origin for public routes.  | `https://*.example.com,http://localhost:*`                              |
-| `ALLOWED_ORIGINS_CLIENTS` (optional)                     | List of urls allowed as origin for clients routes. | `https://*.example.com,http://localhost:*`                              |
-| `REFARCH_SECURITY_CSRFWHITELISTED_<index>` (optional)    | List of routes to disable csrf protection for.     | `/example/**`                                                           |
+| Var                                                      | Description                                                       | Example                                                                 |
+|----------------------------------------------------------|-------------------------------------------------------------------|-------------------------------------------------------------------------|
```

Comment:
vielleicht ist es auch übersichtlicher irgendwie Unterüberschriften einziehen, sodass man die Konfigs gruppiert nach Funktion

Reply:
Think that's a good idea in general but didn't come up with any good splitting.
Think a heading for a single property is more unreadable as helping.